python/tvm/relay/dataflow_pattern/__init__.py

```diff
@@ -112,21 +112,23 @@ def match(self, expr: Expr) -> bool:
         """
         return match(self, expr)
 
-    def partition(self, expr: Expr) -> bool:
+    def partition(self, expr: Expr, attrs=None) -> Expr:
         """
         Parition the expression into functions defined by this pattern
 
         Parameters
         ----------
         expr : tvm.relay.Expr
             The expression to match.
+        attrs : dict[str->Object]
```

Comment:
Just carious. Is this also a valid type format in Python 3.6+? Cuz I only know `Dict[str, Object]` in this case.

Also since this could also be None, it should be `Optional[Dict[str, Object]]` or `dict[str->Object] or None` I guess?

Reply:
Changed